content/en/blog/2024/2024-01-resolutions-passed.md

```diff
@@ -0,0 +1,85 @@
+---
+title: "Resolutions passed"
+linkTitle: Resolutions passed
+date: 2024-01-31
+author: dma ([@dma](https://hachyderm.io/@dma))
+description: Recent resolutions passed by the board
+---
+
+Since [losing our founding member and President, Kris Nóva](https://nivenly.org/blog/2023/08/19/an-announcement-regarding-kris-n%C3%B3va/),
+we have struggled to get together as a board to pass some much needed 
+resolutions.  Our secretary, Preston Doster, developed an alternative approach
+that has allowed us to pass the most critical resolutions with an offline
+unanimous vote.  It is appropriate then that our first post of the New Year will
+be one announcing our new resolutions.
+
+## Recognition of Kris Nóva
+
+> **WHEREAS** it is deemed desirable and in the best interests of this 
+> organization that the following actions
+> be taken by the Directors of this organization pursuant to this Unanimous 
+> Written Consent;
+> 
+> **WHEREAS**, the Nivenly Foundation, the broader open source software 
+> community, and the world lost a beloved, valued member and leader with the 
+> death of Kris Nóva;
+> 
+> **WHEREAS**, Nóva was a constant champion of inclusion, diversity, and love 
+> for all people;
+> 
+> **WHEREAS**, her numerous contributions to open source software projects have 
+> greatly influenced and shaped the landscape of the modern software industry;
+> 
+> **WHEREAS**, she was a tireless and vocal leader for transgender acceptance, 
+> rights, and equality;
+> 
+> **NOW, THEREFORE, BE IT RESOLVED**, that, pursuant to applicable law, the 
+> undersigned, being all of the Directors of this organization, hereby consent 
+> to, approve, and adopt the following:
+> 
+> (1) On behalf of its members, recognize the vast contributions of Kris Nóva 
+> over the course of her distinguished career and life; and
+> 
+> (2) Extend its sincere condolences to Kris Nóva’s family and friends over the 
+> loss of an amazing human;
+
+## Nivenly as 501(c)(4)
+
+> **WHEREAS**, it is deemed desirable and in the best interests of this 
+> organization that the following actions be taken by the Directors of this 
+> organization pursuant to this Unanimous Written Consent:
+> 
+> **NOW, THEREFORE, BE IT RESOLVED** that, pursuant to applicable law, the 
+> undersigned, being all of the Directors of this organization, hereby consent 
+> to, approve, and adopt the following:
+> 
+> * Registration of the Nivenly Foundation as a “social welfare organization” 
+> under 501(c)(4) organization under the United States IRS Tax Code.
+
+After discussing the possibility of various options for registering the Nivenly 
+Foundation, including 501(c)(3) and 501(c)(4), we have decided to register as a
+501(c)(4), a "social welfare organization".  This means that donations are not 
+tax-deductible, however it operates much the same as a charitable organization 
+otherwise.  It also allows us, should we want to in future, to engage in 
+lobbying and support candidates for office.
```

Comment:
While true, we should probably make a statement that this isn't in our interests or not bring it up ("engage in lobbying"). Otherwise most people won't read it as a neutral communication that just states what we're allowed to do, but could instigate an undue amount of "why would they mention this if they weren't doing it? see! I knew when they started with AI!" etc.